auth/src/main/resources/application.yml

```diff
@@ -1,26 +1,28 @@
 spring:
   application:
     name: "auth"
-  profiles:
```

Comment:
Isn't this how we would want to handle this in the other environments?  Fine if we're punting for now...

Reply:
profiles seemed to be a nice way to have different beans injected via environment but now the project is so simple that it doesn't require it.